app/models/work.rb

```diff
@@ -0,0 +1,32 @@
+class Work < ApplicationRecord
+
+  # Validations
+  validates :title, presence: true
+  validates :category, presence: true
+  validates :creator, presence: true
+  validates :publication_year, presence: true
+  validates :description, presence: true
+
+  # Relationships
+  has_many :votes, dependent: :destroy
+  has_many :users, :through => :votes
+
+  # Methods
+  def self.spotlight
+    return spotlight_media = Work.order("votes_count DESC").first
+  end
+
+  def self.top_ten(category)
+    return nil if (Work.all).empty?
+
+    top_ten_list = Work.where(category: category).sort_by{|work| [-work.votes_count, work.title]}.first(10)
```

Comment:
You can use `order` here as well:

```suggestion
    top_ten_list = Work.where(category: category).order(votes_count: :desc, title: :asc).first(10)
```